pages/index.vue

```diff
@@ -41,62 +28,8 @@ import { GeneralLink } from '~/constants/appLinks'
 export default class LandingPage extends QiskitPage {
   routeName = 'qiskit-landing-page'
 
-  mosaicSectionTitle = 'What can Qiskit do'
-  mosaicSectionDescription = `Qiskit accelerates the development of quantum applications by
+  qiskitCapabilitiesSectionTitle = 'What Can Qiskit Do'
+  qiskitCapabilitiesSectionDescription = `Qiskit accelerates the development of quantum applications by
     providing the complete set of tools needed for interacting with quantum systems and simulators.`
-  mosaicSectionElements = [
-    {
-      position: 'first',
-      title: 'Access to circuits',
-      description: 'Access a rich set of well-studied circuits, which can be used as benchmarks, building blocks in more complex circuits, or as a tool to explore quantum computational advantage.',
-      image: '/images/landing-page/feature-circuit.png'
-    },
-    {
-      position: 'second',
-      title: 'Hardware Access',
-      description: 'Execute code on multiple quantum hardware architectures, from superconducting qubits to trapped-ions.',
-      image: '/images/landing-page/feature-hardware.jpg'
-    },
-    {
-      position: 'third',
-      title: 'Quantum Algorithms',
-      description: 'Research and prototype machine learning, optimization, and chemistry applications by building upon a library of quantum algorithms.',
-      image: '/images/landing-page/feature-quantum-algorithms.png'
-    },
-    {
-      position: 'fourth',
-      title: 'Noise Mitigation',
-      description: 'Study and reduce the impact of noise using built-in modules for noise characterization and circuit optimization.',
-      image: '/images/landing-page/feature-noise-mitigation.png'
-    }
-  ]
-
-  fullOverviewLink: GeneralLink = {
-    url: '/overview',
-    label: 'Full overview',
-    segment: { cta: 'overview', location: 'mosaic' }
-  }
 }
 </script>
-
-<style lang="scss">
-.landing-page {
-  &__grid {
-    @include responsive-grid-bg-strip('/images/grid/grid-mosaic.svg', 112rem, auto, 16rem);
```

Comment:
We should also delete the file in **static/images/grid/grid-mosaic.svg** as it's no longer used.

Reply:
Thanks for the review, @eddybrando. I'll wait for confirmation from Russell before I finalize the deletions...

Reply:
Apologies - this is correct, in that it's an unused file. Deleted.